data/wai-aria11.json

```diff
@@ -1,3 +1,6 @@
 {
+  "impl": {
+    "chromestatus": 5761503818940416
```

Comment:
Since we're on it, could you also add a link to the information on edgestatus?

```json
"edgestatus": "Accessible Rich Internet Applications (WAI-ARIA) 1.1"
```

See: https://developer.microsoft.com/en-us/microsoft-edge/platform/status/accessiblerichinternetapplicationswaiaria11/

Reply:
Good catch. Thanks!